src/pages/games/[id]/edit.tsx

```diff
@@ -8,7 +8,7 @@ import React from "react";
 import DeleteGameComponent from "@/components/GameComponent/DeleteGameComponent";
 import { populatedGameWithId } from "@/server/db/models/GameModel";
 import { useSession } from "next-auth/react";
-
+//Comment for ticket
```

Comment:
remove

Reply:
I'm a bit confused, what do you want me to remove specifically in this comment?

Reply:
sorry I should've been more clear -- I just meant remove the comment on line 11.